dLux/dev/tests/apertures.py

```diff
@@ -0,0 +1,13 @@
+import sys
+sys.path.append("dLux/dev")
+from layers import *
+
+help(SoftEdgedCircularAperture)
+dir(SoftEdgedCircularAperture)
+
+aperture = SoftEdgedCircularAperture(1024, .25, -.25, np.pi / 4., 0., 1., 1. / 1024., .5)
+# aperture = SoftEdgedCircularAperture(1024)
+
+pyplot.imshow(aperture._aperture())
+pyplot.colorbar()
+pyplot.show()
```

Comment:
Is this script meant to be here? Not sure if its actually doing anything

Reply:
I just used it for testing all of the different apertures and their soft edged counter parts by hand. It can be left of the pull request.